### DISCUSSION TITLE
Smart reply generation

### DISCUSSION THREAD

**Discussion by adar2378** [invertase/react-native-firebase#5633] [General]
Hi, I've been trying to integrate firebase nlp based smart replies. But it looks like this package is deprecated and doesn't go along with latest version of firebase sdk https://www.npmjs.com/package/@react-native-firebase/ml-natural-language

And on the other hand the react-native-mlkit library shows not found. https://github.com/invertase/react-native-mlkit

Could anyone help me with this? How should I implement smart replies? Thanks in advance.

**Comment by mikehardy** [comment]
If you need ML features you'll have to either use an old version before this breaking change in v12 https://github.com/invertase/react-native-firebase/blob/master/CHANGELOG.md#1200-2021-05-19 (v11.5.0 was the last version there that had it) or wait for us to get the mlkit code from that version out in it's own library (might be a while)

Note that the firebase team put a migration document up - I believe the idea was to wrap the features in cloud functions and call the cloud. Not quite as slick as on-device inference but we don't have the new google MLKit wrapped for react-native (yet)
